modules/platforms/dotnet/Apache.Ignite.Core/Transactions/ITransactions.cs

```diff
@@ -104,6 +104,18 @@ public interface ITransactions
 
         /// <summary>
         /// Returns a list of active transactions initiated by this node.
+        ///  <para/>
+        /// Transactions support next operations:
```

Comment:
`Returned transactions support the following operations:`.

Maybe we should make it simpler and say `Returned transactions do not support Commit and CommitAsync operations`?

Reply:
`ThreadId`, `StartTime` and metadatas ones not supported either.

Reply:
Fixed.